src/pages/List.js

```diff
@@ -111,6 +128,9 @@ export default function List({ token }) {
                         }
                       />
                       {doc.data().item_name}
+                      <button key={doc.id} onClick={() => deleteItem(doc.id)}>
+                        Delete
+                      </button>
```

Comment:
Beautiful! Works perfectly. Great job @zeeatwork and @apdsrocha

Reply:
Thanks @jamesncox!!!

Reply:
I couldn't agree more with the awesome work!